go/vt/vtgate/logstats_test.go

```diff
@@ -193,7 +193,8 @@ func TestLogStatsContextHTML(t *testing.T) {
 	}
 	ctx := callinfo.NewContext(context.Background(), callInfo)
 	logStats := NewLogStats(ctx, "test", "sql1", "", map[string]*querypb.BindVariable{})
-	if string(logStats.ContextHTML()) != html {
+	var tmp = string(logStats.ContextHTML())
+	if tmp != html {
```

Comment:
This change seems unnecessary. We are not even using `tmp` to print out the failure message.

Reply:
yeah I did it for debugging purpose .. forgot to revert it ..